package.json

```diff
@@ -58,7 +58,7 @@
     "xhr-mock": "^2.5.1"
   },
   "dependencies": {
-    "@eppo/js-client-sdk-common": "2.2.2",
+    "@eppo/js-client-sdk-common": "file:../js-client-sdk-common",
```

Comment:
This will need to be updated to either the git repo link after [this PR](https://github.com/Eppo-exp/js-client-sdk-common/pull/46) is merged or the published version is available in npm. For now, please ignore the failing automated checks